docs/modules/operation/pages/deep-dive/kafka-producer/configure-kafka.adoc

```diff
@@ -124,9 +124,9 @@ Consult the source code of the `org.opennms.netmgt.model.OnmsAlarm` class in you
 
 To enable metric forwarding, set the value of the `forward.metrics` property to `true`.
 
-WARNING: The producer's code sends a full CollectionSet per message to Kafka.
+The producer's code sends a full CollectionSet per message to Kafka.
```

Comment:
This may be updated to `Kafka producer tries to send full CollectionSet response as one message in Kafka however If the message size is over 1MB , they will broken up into smaller messages that are less than 1MB.  When sending one CollectionSet over multiple messages, their timestamp will remain same.

Reply:
```suggestion
Kafka Producer sends a full CollectionSet as one message to Kafka.
```